ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-presubmits.yaml

```diff
@@ -807,6 +807,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-assisted-ipv6,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Can you change it to false?
So that it will only run when triggered specifically

Reply:
Changed it, thank you!